cmd/cli/version.go

```diff
@@ -96,8 +99,10 @@ func newVersionCmd(out io.Writer) *cobra.Command {
 				latestReleaseVersion, err := getLatestReleaseVersion()
 				if err != nil {
 					multiError = multierror.Append(multiError, fmt.Errorf("Failed to get latest release information: %w", err))
-				} else if err := outputLatestReleaseVersion(versionCmd.out, latestReleaseVersion, cliVersionInfo.Version); err != nil {
-					multiError = multierror.Append(multiError, fmt.Errorf("Failed to output latest release information: %w", err))
+				} else if cliVersionInfo.Version != devVersion {
```

Comment:
The version can be any version string, not necessarily `dev`.

For e.g. the version can be set to `foo` as follows:
```
VERSION=foo make build-osm
```

Why can't we print whatever version is encoded instead of only allowing specific version strings?

Reply:
Great point. I forgot that the version name is customizable. I will try to modify the regex. Thanks!

Reply:
Hey @lam-man, I am thinking we could simply print the version string instead of displaying an error. Would that work?

Reply:
This definitely will work. Should we also print out the latest available version?

Reply:
Sure, as long as it fixes the existing bug where a custom version string gets incorrectly displayed as reported in #5007